apps/shipments/geojson.py

```diff
@@ -34,22 +34,26 @@ def build_line_string_feature(shipment, tracking_data):
 def build_point_features(shipment, tracking_data):
     """
     :param shipment: Shipment to be used for datetime filtering
-    :param tracking_data: List of tracking data points returned from Engine
+    :param tracking_data: queryset of TrackingData objects
     :return: All tracking coordinates each in their own GeoJSON Point Feature
     """
     begin = (shipment.pickup_actual or datetime.min).replace(tzinfo=None)
     end = (shipment.delivery_actual or datetime.max).replace(tzinfo=None)
     LOG.debug(f'Build point features with tracking_data {tracking_data}.')
     log_metric('transmission.info', tags={'method': 'build_point_features', 'module': __name__})
-    point_features = []
+
+    raw_points = []
     for point in tracking_data:
         try:
             dtp = DeviceTrackingPoint(point)
```

Comment:
Actually, now that we have a `TrackingData` model, we really don't have much use for this `DeviceTrackingPoint` class, it's a little redundant. We could move the `DeviceTrackingPoint.as_point`, `as_point_feature`, `get_linestring_list`, `get_linestring_feature` methods to the `TrackingData` model and delete the `DeviceTrackingPoint` class all-together. I think this refactoring could clean up the code a good bit.